WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloads/Downloadable FIle List/ProductDownloadListViewController+Draggable.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+import CoreServices
+import Yosemite
+import WordPressUI
+
+// MARK: - UITableViewDragDelegate
+//
+extension ProductDownloadListViewController: UITableViewDragDelegate {
+    func tableView(_ tableView: UITableView, itemsForBeginning session: UIDragSession, at indexPath: IndexPath) -> [UIDragItem] {
+        let item = viewModel.item(at: indexPath.row)
+        if let item = item {
```

Comment:
Maybe we can merge it:
```suggestion
        if let item = viewModel.item(at: indexPath.row) {
```

Reply:
👍